app/views/logs/index.html.erb

```diff
@@ -64,9 +64,24 @@
   <%= render partial: "log_filters" %>
 
   <div class="app-filter-layout__content">
-    <%= render SearchComponent.new(current_user:, search_label: "Search by log ID, tenant code, property reference or postcode", value: @searched) %>
+    <% if current_page?(controller: 'lettings_logs', action: 'index') %>
+      <%= render SearchComponent.new(current_user:, search_label: "Search by log ID, tenant code, property reference or postcode", value: @searched) %>
+    <% end %>
+
+    <% if FeatureToggle.sales_log_enabled? && current_page?(controller: 'sales_logs', action: 'index') %>
+      <%= render SearchComponent.new(current_user:, search_label: "Search by log ID, purchaser code or postcode", value: @searched) %>
```

Comment:
Could we extract search_label into a method (maybe in a helper?) and lose the if statements in the erb?

Reply:
Good idea :D